zerver/actions/users.py

```diff
@@ -589,12 +590,9 @@ def send_stream_events_for_role_update(
         now_inaccessible_streams = [
             stream for stream in old_accessible_streams if stream.id in now_inaccessible_stream_ids
         ]
-        event = dict(
-            type="stream",
-            op="delete",
-            streams=[stream_to_dict(stream) for stream in now_inaccessible_streams],
+        send_stream_deletion_event(
+            realm=user_profile.realm, users=[user_profile.id], streams=now_inaccessible_streams
```

Comment:
There is no need to have named arguments here, it is already clear on what values we are passing.